README.md

```diff
@@ -330,12 +330,12 @@ hardware (as most share trigger out and sync out pins).
 The hardware configuration file is ``config/trigger_pins.yaml``. The
 mappings for ``hal_plugin_gen*`` come from
 [Prophesee
-documentation](https://docs.prophesee.ai/stable/metavision_sdk/modules/metavision_hal/samples/hal_viewer.html#accessing-triggers).
+documentation](https://docs.prophesee.ai/3.1.2/metavision_sdk/modules/hal/samples/hal_viewer.html#configuring-trigger-in-and-trigger-out).
```

Comment:
@berndpfrommer since MV SDK 4.0 there is no more "pin mapping" in the API for the Trigger In configuration. The new metavision_hal_viewer (renamed metavision_hal_showcase) shows the new API in action: https://docs.prophesee.ai/stable/samples/modules/hal/hal_showcase.html#configuring-trigger-in-and-trigger-out

As your repo/code relies on this pin mapping, I updated the link to point to the previous version of this API. If/when you update your repo to use our 4.x API then you can update the link to the "stable" URL.